bedrock/mozorg/templates/mozorg/about-base.html

```diff
@@ -38,9 +39,19 @@ <h4>{{ _('Get Involved') }}</h4>
 {% endblock %}
 
 {% block email_form %}
-  {% if LANG.startswith('en-') %}
-    {{ email_newsletter_form(newsletters='mozilla-foundation', title=_('Love the Web?'), subtitle=_('Get the Mozilla newsletter and help us keep it open and free.'), button_class='button-hollow button-light', spinner_color='#fff') }}
-  {% else %}
-    {{ email_newsletter_form(button_class='button-hollow button-light', spinner_color='#fff') }}
-  {% endif %}
+<div class="mzp-l-content">
+  <aside class="mzp-c-newsletter t-love">
+    <div class="mzp-c-newsletter-image">
+      <img src="{{ static('img/newsletter/love-news.svg') }}" alt="" height="249" width="300">
```

Comment:
The newsletter icon feels pretty huge - could it maybe be a touch smaller?

![image](https://user-images.githubusercontent.com/400117/60732592-5d17a780-9f42-11e9-8e77-de1770c107e2.png)